python/ray/tune/tune.py

```diff
@@ -141,6 +141,19 @@ def run_experiments(experiments,
     if restore:
         runner = try_restore_runner(checkpoint_dir, search_alg, scheduler,
                                     trial_executor)
+
+        runner._trials[0].checkpoint_freq = experiments[0].spec['checkpoint_freq']
+
+        for key in ['grad_clip', 'lr', 'entropy_coeff', 'vf_loss_coeff', 'monitor']:
```

Comment:
don't we want to copy whole config from experiments[0].spec['config']?

Reply:
not all flags are safe to copy since they may affect the build up of the model (we want to keep the model structure that was checkpointed), such as model.custom_model flag, model.custom_options.command_encoding, etc.

Reply:
which reminds me to remove command_encoding flag from being overridden (it was just temporary there)

Reply:
changed a lot here, take a look.